packages/syntax-core/src/Box/Box.tsx

```diff
@@ -98,6 +98,14 @@ type BoxProps = {
    * The background color of the box.
    */
   backgroundColor?: (typeof allColors)[number];
+  /**
+   * The border color of the box.
+   * This is a passthrough prop directly to the Box component.
+   *
+   * Usage example: `border="1px solid var(--color-base-gray-300)"`
+   * @defaultValue `none`
+   */
+  border?: string;
```

Comment:
I'm fully aware this is terrible, I was wondering if @christianvuerings had a better way of typing this?